cmd/zc_traverser_blobfs_account.go

```diff
@@ -0,0 +1,102 @@
+// Copyright © 2019 Microsoft <wastore@microsoft.com>
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package cmd
+
+import (
+	"context"
+	"errors"
+	"net/url"
+	"path/filepath"
+
+	"github.com/Azure/azure-pipeline-go/pipeline"
+
+	"github.com/Azure/azure-storage-azcopy/azbfs"
+)
+
+// We don't allow S2S from BlobFS, but what this gives us is the ability for users to download entire accounts at once.
```

Comment:
Good thinking to make things consistent in this way.  Is there a smoke test case written for this?  (There needs to be, esp since its close enough to a niche usage that it won't naturally get covered in load tests, manual tests etc)

Reply:
As these will get integrated into initResourceTraverser during the S2S refactor, I'll include smoke testing for that. As is, there is unit testing for the traversers.